controllers/articles.controllers.js

```diff
@@ -20,3 +22,17 @@ exports.getArticles = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.patchArticleByIdVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  Promise.all([
+    updateArticleByIdVotes(article_id, inc_votes),
```

Comment:
this is up to you, you could have your Promise.reject from your checkArticleExists in your updateArticle instead and then you woudlnt need promise.all